__tests__/api.test.js

```diff
@@ -136,5 +136,76 @@ describe ("/api/articles/:article_id/comments", ()=>{
     })
 })
 
+describe("POST /api/articles/:article_id/comments", ()=>{
+    test('POST:201 inserts a new comment for an article to the db and returns the posted comment with correct properties and their values (votes default to 0)', () => {
+            const newComment = {
+            username: 'butter_bridge',
+            body: "I cannot imagine any joy comming out of this cold depressing world"
+            };
+            return request(app)
+            .post('/api/articles/1/comments')
+            .send(newComment)
+            .expect(201)
+            .then((response) => {
+                const comment = response.body.comment
+                expect(comment.article_id).toBe(1)
+                expect(comment.author).toBe("butter_bridge")
+                expect(comment.body).toBe("I cannot imagine any joy comming out of this cold depressing world")
+                expect(comment.votes).toBe(0)
+            });
+        });
```

Comment:
We could add a test for when we have extra properties in the newComment. We should still be able to post and ignore anything extra 🙂